userspace/libsinsp/event.cpp

```diff
@@ -738,7 +738,7 @@ char* sinsp_evt::render_fd(int64_t fd, const char** resolved_str, sinsp_evt::par
 
 Json::Value sinsp_evt::get_param_as_json(uint32_t id, OUT const char** resolved_str, sinsp_evt::param_fmt fmt)
 {
-	ASSERT(id < m_info->nparams);
+	ASSERT(id < (m_info->nparams < m_pevt->nparams ? m_info->nparams : m_pevt->nparams));
```

Comment:
Now that this part is a little bit more complicated, it might be worth moving the assert down below after `load_params()` and simplifying it with `ASSERT(id < get_num_params())`?

Reply:
I agree, that's better, will do